### DISCUSSION TITLE
Timeout error while extracting interfaces information

### DISCUSSION THREAD

**Discussion by amrchnk** [scrapli/scrapligo#159] [Q&A]
Hello! Sorry if my question looks stupid, but I can't find solution. I use scrapligo lib to get info about devices in my subnet, and It works pretty cool with cisco routers, but when I try to scrap info about device interfaces from switches with command `show interfaces`, It always falls down with timeout error even though timeouts are increased, at the same time another commands like `show version` work nice. What could be wrong?
I try to use basic generic/network drivers from examples.

**Comment by carlmontanari** [comment]
without logs or any information pretty impossible to say, especially not knowing what platform(s) you're trying to connect to. good bet is you are not disabling paging or whatever the equivalent is for your platforms.
